### DISCUSSION TITLE
ValueError: 'readlive' is not a valid PermissionNode

### DISCUSSION THREAD

**Discussion by jesperprintz** [ep1cman/unifi-protect-backup#138] [General]
Hi.

I installed unifi-protect-backup-0.10.7 via PIP.
But can see it has stopped working, when i see in syslog i get this output from python.

ValueError: 'readlive' is not a valid PermissionNode

Is it a bug or has some config changed i cant see?

**Comment by ep1cman** [comment]
You will need to update the version of pyunifiprotect you have installed, I should probably set the minimum required version to something newer

**Comment by ep1cman** [comment]
Can you post a full log of the error

**Comment by jesperprintz** [comment]
sorry i thought i managed to delete my comment.

Was my own mistake, i ran the application as a user, but updated pyunifiprotect  as root, so the one i was using was not actually being updated,

**Comment by michalek-b** [comment]
How to update script in running docker container?

**Reply by ep1cman** [reply]
> How to update script in running docker container?

You don't, you update the container instead
